app/views/articles/index.html.erb

```diff
@@ -157,6 +157,6 @@
     <%= render "articles/sidebar_additional" %>
   </div>
 
-  <%= render "stories/narrow_nav_menu" %>
+  <div class="test"><%= render "stories/narrow_nav_menu" %></div>
```

Comment:
class test, is this right?

Reply:
oh shoot, no :D . good catch, i'll remove it!